actiontext/app/javascript/actiontext/attachment_upload.js

```diff
@@ -35,6 +40,18 @@ export class AttachmentUpload {
       .replace(":filename", encodeURIComponent(filename))
   }
 
+  dispatch(name, detail = {}) {
+    detail.attachment = this.attachment
+    return dispatchEvent(this.element, `direct-upload:${name}`, { detail })
+  }
+
+  dispatchError(error) {
+    const event = this.dispatch("error", { error })
+    if (!event.defaultPrevented) {
+      throw new Error(`Direct upload failed: ${error}`)
```

Comment:
@rafaelfranca Action Text throws an error (which is unrecoverable from the Rails app) while Active Storage uses an `alert`. I didn't want to do too much in this PR, but do you think we should update to an `alert` instead of "silently" failing?

Reply:
@rafaelfranca is it okay to change this error to an alert? I'm unsure if that qualifies as a braking change or we want to keep the previous behavior

Reply:
Let's do that.

Reply:
Done! I've also updated the docs.